### DISCUSSION TITLE
A new Low-Code platform using BootstrapVue

### DISCUSSION THREAD

**Discussion by renaudpawlak** [bootstrap-vue/bootstrap-vue#6964] [Show and tell]
Hope it's not too off-topic but I would like to point out my [DLite](https://github.com/cincheo/dlite) Open Source project, which uses BootstrapVue. It is a low-code development tool that wraps BootstrapVue components in an on-line IDE. You can just drag and drop BootstrapVue components (and other components) in your app and get started instantly without having to write any code (or very little).

In available components, you will find most of the BootstrapVue components with most properties available, although some are still missing and/or have been simplified.  

It also has some cool features like UI generation from JSON models, local-first API (data sync), and resource monitoring for eco-design. Feel free to comment or ask questions if any :)

BTW, if you are just curious to drag and drop components, you can try it directly here: https://platform.dlite.io/ (create a new project button)

**Comment by hasen666** [comment]
hi renaud,

this is a very interesting new!
